### PR TITLE
Address user comments and issues 177, 791, and 4109

### PR DESCRIPTION
**Pull request by kraigb**
Also updates for the VS2017 .NET Standard walkthrough to match current Visual Studio 2017 RC.4.

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Consume-Packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Consume-Packages/Finding-and-Choosing-Packages.md) | :warning:Warning | | [Details](#user-content-docs/Consume-Packages/Finding-and-Choosing-Packages.md)

<a name="docs/Consume-Packages/Finding-and-Choosing-Packages.md"></a>
### [docs/Consume-Packages/Finding-and-Choosing-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/kraigb/docs/Consume-Packages/Finding-and-Choosing-Packages.md)
 - **Line 59**: **[Warning]** Invalid file link:(~/create-packages/supporting-multiple-target-framework.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201702082212544942-185%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201702082216363201-185%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Schema/msbuild-targets.md [hunk 1]
```diff
@@ -268,58 +268,6 @@ Restore creates the following files in the build `obj` folder:
 | {projectName}.projectFileExtension.nuget.g.props | References to MSBuild targets contained in packages |
 | {projectName}.projectFileExtension.nuget.g.targets | References to MSBuild props contained in packages |
```
Comment (emgarten) [inline docs/Schema/msbuild-targets.md]: Looks like these got reversed, .targets says props

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C2%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201702132340511119-185%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**